src/app/store/userSettingsSlice.ts

```diff
@@ -42,6 +41,7 @@ const DEFAULT_APPLICATION_MEDIA_SEARCH_WALL_SORT_DIRECTION = "ascending";
 const DEFAULT_APPLICATION_MEDIA_SEARCH_WALL_VIEW_MODE = "cards";
 const DEFAULT_APPLICATION_THEME = "dark";
 const DEFAULT_APPLICATION_USE_IMAGE_BACKGROUND = true;
+const DEFAULT_APPLICATION_VOLUME_LIMIT = null;
```

Comment:
I'm happy to keep `null` as the default here. I did notice one minor downside when testing though: For amplifiers which support a volume limit, the user initially sees an empty value (which makes sense). If they then set a value, it becomes a number (between 0 and their volume limit). Once a value has been set, it's not possible to return to the original `null` state, which means the user then needs to commit to setting a value from that point forward.

Not a big deal, and not worth fixing. I just wanted to point out the UX consequence while I think of it.

In the future, user settings should probably be rethought and perhaps moved out of the "Status" section entirely (settings were wedged into "Status" just for a place to put them originally). At that time, more thought can go into the UX for this stuff. It might benefit from a pattern to represent an optionally-set value.

Reply:
> Once a value has been set, it's not possible to return to the original null state, which means the user then needs to commit to setting a value from that point forward.

You can just clear out the field. If that's unintuitive, maybe we add a trashcan icon or something to explicitly clear it?

Reply:
Oh, my bad. This is totally fine as-is and works as described. Thanks for pointing that out!

(I do think it would be nice one day to have a separate User Settings section with clearer patterns for optional settings like this one; but that's for another day).